deploy/charts/harvester/README.md

```diff
@@ -1,108 +1,58 @@
+# Harvester Helm Chart
 
 ## Chart Details
 
 This chart will do the following:
 
 - Deploy a KubeVirt Operator if needed, defaults to deploy.
-- Deploy a KubeVirt CRD resource to enable KubeVirt if needed, defaults to deploy.
 - Deploy the Harvester resources.
 - Deploy [Longhorn](https://longhorn.io) as the built-in storage management.
+- Deploy Containerized Data Importer (CDI) as supporting third-party storage.
+- Deploy csi-snapshotter
+- Deploy Snapshot Validation Webhook
+- Deploy whereabouts
 
-### Prerequisites
-
-- Kubernetes 1.16+.
-- Helm 3.2+.
-- [Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni). An [example chart](https://github.com/rancher/rke2-charts/tree/main-source/packages/rke2-multus/charts).
-- [Harvester CRDs](https://github.com/harvester/harvester/tree/master/deploy/charts/harvester-crd)
-
-### Installing the Chart
-
-To install the chart with the release name `harvester`.
-
-```bash
-## create target namespace
-$ kubectl create ns harvester-system
-
-## install chart to target namespace
-$ helm install harvester harvester --namespace harvester-system --set service.harvester.type=NodePort
-```
-
-### Uninstalling the Chart
-
-To uninstall/delete the `harvester` release.
-
-```bash
-## uninstall chart from target namespace
-$ helm uninstall harvester --namespace harvester-system
-```
-
-#### Notes
-
-- Use the existing KubeVirt/Longhorn Service.
+## Introduction
 
-    If you have already prepared the KubeVirt or Longhorn, you can disable these installations in this chart.
-    
-    ```bash
-    $ helm install harvester harvester --namespace harvester-system \
-        --set kubevirt.enabled=false --set kubevirt-operator.enabled=false \
-        --set longhorn.enabled=false
-    ```
+Harvester uses a `ManagedChart` resource to deploy its core components. This is configured via the [harvester-installer](https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-10-harvester.yaml). Be aware that `deploy/harvester` and `deploy/harvester-crd` are two separate `ManagedChart` resources.
 
-- Use other storage drivers.
-    
-    Currently, storage drivers other than Longhorn are not supported.
+Once a `ManagedChart` is created, Fleet generates a `Bundle` resource to trigger Helm and install the chart. You can learn more about this process in the [Fleet documentation](https://fleet.rancher.io/concepts).
 
-### Configuration
+During upgrades, Harvester runs a script to patch the release version in the `ManagedChart`. This happens in the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L885-L935) script.
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install` or `helm upgrade`.
+> If you're interested in this change, please check [this issue](https://github.com/harvester/harvester/issues/8163) for more details.
```

Comment:
I don't think linking to this issue is necessary in the intro section. Feels more relevant in the CRD chart.

Reply:
Agree. I'll remove that.